### PR TITLE
added contributing.md

### PR DESCRIPTION
**Pull request by rohansharmasitoula**
Resolves: https://github.com/talview/moodle-quizaccess_proctor/issues/18

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (devang1281) [inline CODE_OF_CONDUCT.md:63]: @rakeshprabhu Please guide what contact details should be updated

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+# Contributing to Talview Proview Moodle (Quizaccess-Proctor) Plugin
```
Comment (devang1281) [inline CONTRIBUTING.md:1]: @rohansharmasitoula Update this to Proview - Moodle Plugin (quizaccess_proctor)

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (rohansharmasitoula) [reply CODE_OF_CONDUCT.md]: Added

File: CONTRIBUTING.md (hunk 2 above)
Comment (rohansharmasitoula) [reply CONTRIBUTING.md]: Updated

**Review by devang1281 — APPROVED** [review]
LGTM

**Pull request merged**